Sources/TokamakDOM/DOMNode.swift

```diff
@@ -78,106 +111,90 @@ extension AnyHTML {
       } else {
         _ = dom.ref.setAttribute!(attribute.value, value)
       }
-
-      if attribute == .checked {
-        containsChecked = true
-      }
     }
+  }
 
-    // Animate styles
-    if let style = attributes["style"],
-       let animation = transaction.animation
-    {
-      let resolved = _AnimationProxy(animation).resolve()
-      func extractStyles(compute: Bool = false) -> [String: String] {
-        var res = [String: String]()
-        let computedStyle = JSObject.global.getComputedStyle?(dom.ref)
-        for i in 0..<Int(dom.ref.style.object?.length.number ?? 0) {
-          guard let key = dom.ref.style.object?[i].string else { continue }
-          if compute {
-            res[key] = computedStyle?[dynamicMember: key].string
-              ?? dom.ref.style.object?[key].string
-          } else {
-            res[key] = dom.ref.style.object?[key].string
-          }
+  private func animateStyles(
+    _ style: String,
+    on dom: DOMNode,
+    with animation: Animation
+  ) {
```

Comment:
I realize these linter warnings are a bit annoying, but is it still possible to get it a bit slimmer and under 50 lines? Maybe would work better from code readability perspective too.